persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/SampleListMyBatisRepositoryTest.java

```diff
@@ -80,8 +80,8 @@ public void getAllSampleListsDetailedProjection() throws Exception {
             " Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href=\\\"http://tcga-data.nci." +
             "nih.gov/tcga/\\\">Raw data via the TCGA Data Portal</a>.", cancerStudy.getDescription());
         Assert.assertEquals(true, cancerStudy.getPublicStudy());
-        Assert.assertEquals("23000897,26451490", cancerStudy.getPmid());
```

Comment:
same comment here as above regarding multiple PMIDs for a study

Reply:
also as part of that change, the citation now ends with `, ...` if there are multiple PMIDs for a study